src/nnetbin/cuda-gpu-available.cc

```diff
@@ -41,6 +53,34 @@ int main(int argc, char *argv[]) try {
   CuDevice::Instantiate().SelectGpuId("yes");
   std::cerr
     << "### HURRAY, WE GOT A CUDA GPU FOR COMPUTATION!!! ###"
+    << std::endl << std::endl;
+  std::cerr
```

Comment:
There is a lot of std::cerr here.
Can you please use KALDI_LOG or KALDI_WARN  or KALDI_ERR at least some of the time, so that
the version-number info and the name of the binary will be printed?
It's fine with multi-line error messages.